<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>543</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>通道索引：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>设备：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="DeviceCombox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>USBCAN1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCAN2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PCI-9820I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCAN_E_U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCAN_2E_U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCAN_4E_U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PCIECANFD-100U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PCIECANFD-200U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PCIECANFD-400U-EX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCANFD-200U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCANFD-100U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCANFD-MINI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANET-TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANET-UDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANWIFI-TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANWIFI-UDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CLOUD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDWIFI-100U-TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDWIFI-100U-UDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDNET-200U-TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDNET-200U-UDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDNET-400U-TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFDNET-400U-UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设备索引：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="DeviceIndexCombox"/>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="ChannelIndexCombox"/>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>631</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>631</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>模式：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="ModelCombox">
        <item>
         <property name="text">
          <string>正常</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>只听</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QCheckBox" name="TerminalCheckBox">
        <property name="text">
         <string>终端电阻使能</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" colspan="2">
       <widget class="QComboBox" name="BaudCombox">
        <item>
         <property name="text">
          <string>1Mbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>125kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5kbps</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>滤波模式：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QComboBox" name="FilterModelCombox">
        <item>
         <property name="text">
          <string>双滤波</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单滤波</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>数据域波特率：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" colspan="2">
       <widget class="QComboBox" name="DataBaudCombox">
        <item>
         <property name="text">
          <string>5Mbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4Mbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2Mbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1Mbps</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>验收码(0x)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QLineEdit" name="AcceptanceEdit">
        <property name="text">
         <string>00000000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>屏蔽码(0x)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7" colspan="3">
       <widget class="QLineEdit" name="ShieldEdit">
        <property name="text">
         <string>FFFFFFFF</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>仲裁域波特率：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="ArbitBaudCombox">
        <item>
         <property name="text">
          <string>1Mbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>125kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50kbps</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>131</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>631</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="2">
       <widget class="QPushButton" name="StartCANButton">
        <property name="text">
         <string>启动CAN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_47">
        <property name="text">
         <string>设备控制</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="InitCANButton">
        <property name="text">
         <string>初始化CAN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="ResetButton">
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="OpenDeviceButton">
        <property name="text">
         <string>打开设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="CloseDeviceButton">
        <property name="text">
         <string>关闭设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>631</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>631</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>数据发送</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ID(0x)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>帧类型：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="3">
       <widget class="QComboBox" name="FrameTypeCombox">
        <item>
         <property name="text">
          <string>标准帧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扩展帧</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>CANFD加速：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="13">
       <widget class="QComboBox" name="CANFDAccCombox">
        <item>
         <property name="text">
          <string>是</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>否</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="6" colspan="7">
       <widget class="QLineEdit" name="OnceSendFrameEdit"/>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>一次性发送帧数：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>发送方式：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="SendMannerCombox">
        <item>
         <property name="text">
          <string>正常发送</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单次发送</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>自发自收</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>延时(ms)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="DelayEdit"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="IdEdit"/>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>协议：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>数据(0x)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9" colspan="5">
       <widget class="QLineEdit" name="DataEdit">
        <property name="text">
         <string>00 00 00 00 00 00 00 00 00 00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9" colspan="3">
       <widget class="QComboBox" name="AgreementCombox">
        <item>
         <property name="text">
          <string>CAN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CANFD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="3">
       <widget class="QCheckBox" name="QueueFrameDelayCkb">
        <property name="text">
         <string>队列帧延时标记</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>631</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>队列发送</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="QueueSendModelCkb">
        <property name="text">
         <string>队列发送模式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="GetQueueAvaiButon">
        <property name="text">
         <string>获取队列可用空间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ClearQueueButton">
        <property name="text">
         <string>清除队列</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="FindDeviceModelButton">
        <property name="text">
         <string>查询设备模式</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>索引：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="IndexSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>定时间隔(ms)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLineEdit" name="IntervalTimeEdit"/>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QPushButton" name="AddCANButton">
        <property name="text">
         <string>添加CAN(FD)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="OpenIntervalSendButton">
        <property name="text">
         <string>开启定时发送</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="CloseIntervalSendButton">
        <property name="text">
         <string>关闭定时发送</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="SelectTimeListBtn">
        <property name="text">
         <string>查询定时列表</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="CancelSingleSendBtn">
        <property name="text">
         <string>取消单条定时发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>631</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>631</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>数据接收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="ClearDataButton">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTextBrowser" name="DataRectextBrowser"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
